lib/RNWatch.d.ts

```diff
@@ -1 +1,2 @@
 export * from './RNWatch.ios';
+export * from './RNWatch.android';
```

Comment:
I don't think this will be necessary - this is just a typings file. `useApplicationContext` etc are already defined by `RNWatch.ios` and we could potentially end up accidently overriding types in the future.

Reply:
Please do check though!

Reply:
Yep you were correct, it was not needed